src/commands/install_mod.rs

```diff
@@ -0,0 +1,16 @@
+use crate::mods::ValheimMod;
+use clap::ArgMatches;
+use log::{debug, error, info};
+
+pub fn invoke(args: &ArgMatches) {
+  let mut valheim_mod = ValheimMod::new(args.value_of("URL").unwrap());
+  info!("Installing {}", valheim_mod.url);
+  debug!("Mod URL: {}", valheim_mod.url);
+  debug!("Mod staging location: {}", valheim_mod.staging_location);
+  match valheim_mod.download() {
+    Ok(_) => valheim_mod.install(),
+    Err(message) => {
+      error!("{}", message);
```

Comment:
I figure for a mod, a silent fail is appropriate because in some cases they might already have the mod installed and the url is purely stale. What I could add is it exiting with code 1 on a failure and than in the shell script trapping that failure with a more noticeable message. What are your thoughts?

Reply:
I think that failing within odin and then ignoring that for the shell script would be a good idea! I'll leave the final decision up to you though

Reply:
It looks like all the errors raised by this are from the staging directory and an invalid mod url which seems fair to hard error on both of those, and then it can just be ignored by the container